scalyr_agent/sharded_copying_manager/worker.py

```diff
@@ -1189,6 +1178,15 @@ def get_log_path(self):
         return self.__cached_log_path
 
 
+def change_agent_log(new_path):
```

Comment:
I'm a little worried about relying on the original process to invoke this remote procedure.  Can we instead use `initializer` from the manager's `start` method?  See this documentation https://docs.python.org/2.7/library/multiprocessing.html#multiprocessing.managers.BaseManager.start

It does seem like we can to tell the subprocess to run a specific function upon start up.  It would make more sense to do it then.

Does that work?

Reply:
Sure, thanks for reminding me of that. I completely forgot about this feature. That should be exactly what is needed.